.github/workflows/github-pages-deployment.yml

```
@@ -19,15 +19,16 @@ jobs:
     runs-on: ubuntu-latest
     env:
       GITHUB_PAGES_REPO: https://github.com/cloudposse/docs
-      GITHUB_PAGES_BRANCH: production
+      GITHUB_PAGES_PULL_BRANCH: master
+      GITHUB_PAGES_PUSH_BRANCH: production
       GITHUB_PAGES_DIRECTORY: github_pages
     steps:
     - name: "Checkout source code at current commit"
       uses: actions/checkout@v2
     - name: "Build Hugo Static Site"
-      uses: cloudposse/docs/actions/publish-github-pages@0.64.0
+      uses: cloudposse/docs/actions/publish-github-pages@publish-gh-pages-python
```

```suggestion
      uses: cloudposse/docs/actions/publish-github-pages@publish-gh-pages-python
```
Just a reminder to change this before merging